manubot/process/tests/test_process_command.py

```diff
@@ -27,14 +30,18 @@ def test_example_manuscript(manuscript):
         "--output-directory",
         str(manuscript_dir.joinpath("output")),
     ]
+    if skip_citations:
+        args.append("--skip-citations")
```

Comment:
Do you want to test that skip citations actually skips the citation processing?

Reply:
Not really because I view `manubot process` as a command we're phasing out and test takes time to write. Relying on https://github.com/manubot/rootstock/pull/300 to test `--skip-citations` actually works.

Reply:
`--skip-citations` is working in https://github.com/manubot/rootstock/runs/387673500 / https://github.com/manubot/rootstock/commit/49dcfcdef8c9bf97cc4f33b561657ecec1aea319 which relies on `citation-tags.tsv`.